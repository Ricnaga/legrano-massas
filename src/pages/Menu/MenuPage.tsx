export function MenuPage(){
    return <h1>MenuPage</h1>
}