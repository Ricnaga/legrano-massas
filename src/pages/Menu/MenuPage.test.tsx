import { render } from '@testing-library/react';
import { MenuPage } from './MenuPage';

jest.mock('./components/MenuDoughs', () => ({
  __esModule: true,
  MenuDoughs: () => <div>Mock MenuDoughs</div>,
}));

describe('Page: MenuPage', () => {
  it('should render correctly', () => {
    const { container } = render(<MenuPage />);
    expect(container).toMatchSnapshot();
  });
});
