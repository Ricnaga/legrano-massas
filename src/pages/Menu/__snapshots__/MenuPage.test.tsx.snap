// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MENU SCREEN should render correctly 1`] = `
<div>
  <div
    class="MuiGrid-root css-jn9ra0-MuiGrid-root"
  >
    <div>
      MenuDoughs
    </div>
  </div>
</div>
`;
